### DISCUSSION TITLE
添加新url订阅会报错 for input string:'一串数字'

### DISCUSSION THREAD

**Discussion by Ronin52658** [MetaCubeX/ClashMetaForAndroid#292] [Q&A]
更新订阅和本地导入正常，请问大佬这是什么问题

**Comment by qingmuhy744** [comment]
我也碰到这个问题了，不知道是什么个情况。

![a0a5f660f6c553a029ac688b301c732](https://github.com/user-attachments/assets/293dd8ac-7741-4870-8899-202391485b8b)

**Comment by YuHuanTin** [comment]
同样情况，原因可能是订阅提供的 `Subscription-Userinfo` 字段中出现了小数点的数字，所以导致解析出错
